tests/common/grafana.js

```diff
@@ -0,0 +1,20 @@
+// Available as cy.testGrafanaDashboard("baseUrl", "the names of the Grafana dashboard", "the no of completed api req to wait")
+Cypress.Commands.add('testGrafanaDashboard', (baseUrl, dashboardName, requestsToWait) => {
+  cy.intercept("/api/**").as("api")
+
+  cy.visit(baseUrl + '/dashboards')
+  // used to view and load as much of the dashboard as possible
+  cy.viewport(2560, 2160)
```

Comment:
Would it be possible to maybe extract these numbers as variables somewhere? So that if we wanted to change the default viewport resolution it would be easier?

Reply:
I tried, but didn't manage to make it work
I do not think we will need to change the viewpoint resolution, very often.
The reason I set it:
- for grafana I used the max because I want to load all graphs, and that happens only is they are in view
- for harbor, similar reason, if the viewpoint is too small it will hide some buttons